Features/WalletTab/Sources/Scenes/WalletScene.swift

```diff
@@ -166,16 +172,13 @@ public struct WalletScene: View {
 extension WalletScene {
 
     func refreshable() async {
-        if let walletId = model.keystoreWalletId {
-            Task {
-                do {
-                    try await model.fetch(walletId: walletId, assets: assets)
-                } catch {
-                    NSLog("refreshable error: \(error)")
-                }
+        Task {
+            do {
+                try await model.fetch(walletId: model.wallet.walletId, assets: assets)
```

Comment:
Can we get walletId directly in model?
```suggestion
                try await model.fetch(assets: assets)
```